provisioning/miniprov/hare_mp/main.py

```diff
@@ -37,6 +37,11 @@
 from hare_mp.store import ConfStoreProvider
 from hare_mp.validator import Validator
 
+# Logger details
+LOG_DIR = "/var/log/seagate/hare/"
+LOG_FILE = os.path.join(LOG_DIR, 'setup.log')
```

Comment:
[minor thing]

```suggestion
LOG_FILE = 'setup.log'
```
At the moment where the actual full path is needed, it can be evaluated as follows `os.path.join(LOG_DIR, LOG_FILE)` keeping both directory and filename configurable. Such an evaluation can be formed as a function for readability.

Reply:
done

Reply:
@mssawant Konstantin suggested me to do this minor changes. That's why I done with it.